runtimes/talos-libgdx/src/com/talosvfx/talos/runtime/modules/ColorModule.java

```diff
@@ -80,16 +83,16 @@ public Color getColor() {
     @Override
     public void write (Json json) {
         super.write(json);
-        json.writeValue("r", defaultR);
-        json.writeValue("g", defaultG);
-        json.writeValue("b", defaultB);
+        json.writeValue("color",getColor().toString());
```

Comment:
The serialization strategy here should probably not be changed as it will break all existing projects/particles.

If you'd like the hex string also, its fine to add as a surplus, but please keep the original methods.